File: src/main/java/com/methodpark/cmpe/code/isp/TranslatorFactory.java
package com.methodpark.cmpe.code.isp;

public class TranslatorFactory
{
    public static IFileTranslator createTranslator(Language lang) {
        switch (lang) {
        case de:  return new GermanFileTranslator();
        default:  return null;
        }
      }
}

File: src/main/csharp/Isp/Isp/TranslatorFactory.cs
namespace Isp
{
    public class TranslatorFactory
    {
        public static IFileTranslator CreateTranslator(Language language)
        {
            switch (language)
            {
                case Language.De:
                    return new GermanFileTranslator();
                default:
                    return null;
            }
        }
    }
}